about.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Page</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="aboutstyle.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <section class="mobile-menu hidden">
+        <ul>
+          <li class="cancel">
+            <a href="#">
+              <img src="images/Icon - Cancel.svg" alt="cancel" id="cancel">
+            </a>
+            </li>
+          <li>
+            <a href="index.html">Home</a>
+          </li><hr>
+          <li>
+            <a href="about.html">About</a>
+          </li><hr>
+          <li>
+            <a href="#">Initiatives</a>
+          </li><hr>
+          <li>
+            <a href="#">Resources</a>
+          </li><hr>
+          <li>
+            <a href="#">Contact</a>
+          </li><hr>
+        </ul>
+      </section>
+      <section class="mobile-navigation">
+        <ul>
+          <li>
+            <a href="#"><img src="images/mobile_menu 1.svg" id="menu" alt="menu"></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/fmcde-round-logo.png" id="logo" alt="logo"></a>
+          </li>
+        </ul>
+      </section>
+      <section class="desktop-nav1">
+        <ul>
+          <li>
+            <a href="#">
+              <img src="images/facebook.svg" alt="facebook">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/twitter.svg" alt="twitter">
+            </a>
+          </li>
+          <li>
+            <a href="#">English</a>
+          </li>
+          <li>
+            <a href="#">MyPage</a>
+          </li>
+          <li>
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </section>
+      <section class="desktop-nav2">
+        <img src="images/fmcde-main-logo-trans.png" alt="logo">
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="#">Initiatives</a>
+          </li>
+          <li>
+            <a href="#">Resources</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+          <li>
+            <a href="#" class="news">News</a>
+          </li>
+        </ul>
+      </section>
+    </nav>
+  </header>
+  <main>
+    <section class="main-sect">
+      <section class="title-sec" id="title">
+        <h4 class="orange" id="about-tit">FEDERAL MINISTRY OF</h4>
+        <h1 class="orange" id="about-main-tit">COMMUNICATIONS AND DIGITAL ECONOMY</h1>
+      </section>
```

Comment:
- [ ] Kindly change this `section` tag to `div` tag. For semantics reasons, a `section` tag defines only a section on your page.